submissions/xT0PERx/Popup-style/style.css

```diff
@@ -1,182 +1,162 @@
 /*                                         Верхнее меню с 5 блоками  */
 .container {
-    display: flex;
-    flex-direction: row;
-    justify-content: flex-end;
-    font-size: 1em;
-    padding: 5px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  font-size: 1em;
+  padding: 5px;
 }
 
 .container ol {
-    display: flex;
-    flex-flow: wrap row;
+  display: flex;
 }
 .container li {
-    display: inline-block;
-    width: 50px;
-    height: 50px;
-    margin: 5px;
-    text-align: center;
+  display: flex;
+  width: 50px;
+  height: 50px;
+  margin: 5px;
+  flex-direction: row;
+  align-items: center;
 }
 
 .container div {
-    width: 80px;
-    height: 80px;
-    margin: 5px;
-    cursor: pointer;
+  width: 80px;
+  height: 80px;
+  margin: 5px;
+  cursor: pointer;
 }
 
 .main_tullbar {
-    position: relative;
+  position: relative;
 }
 
 img {
-    width: 32px;
-    height: 32px;
+  width: 32px;
+  height: 32px;
 }
 
 .menu {
-    position: absolute;
-    background-image: url(./images/Menu.png);
-    width: 32px;
-    height: 32px;
-    cursor: pointer;
-    right: 4px;
-    top: 3px;
+  position: absolute;
+  background-image: url(./images/Menu.png);
+  width: 32px;
+  height: 32px;
+  cursor: pointer;
+  right: 4px;
+  top: 3px;
 }
 
 .check_input {
-    -webkit-appearance: none;
+  -webkit-appearance: none;
 }
 
 /*                                                      Normal */
 
-.mail p,
-.images p,
 .menu,
-.menu_more,       
-.home img,
-.avatar img {
-    text-decoration: none;
-    border: 1.5px solid transparent;
+.menu_more,
+.menu_items img {
+  text-decoration: none;
+  border: 1.5px solid transparent;
 }
 
 .menu {
-    border: 1.5px solid white
+  border: 1.5px solid white;
 }
 
-.mail,
-.images,
-.home,
-.avatar {
-    text-decoration: none;
+.menu_items {
+  text-decoration: none;
+  border: 1.5px solid transparent;
 }
-.icons div {
-    border: 1.5px solid transparent;
+.popup_icons {
+  border: 1.5px solid transparent;
+  display: inline-table;
+  text-align: center;
+  margin: 3px;
 }
-
-.icons2 div {
-    border: 1.5px solid transparent;
-}
-
-.check_input,
-.second {
-    opacity: 0;
-    height: 0;
-    width: 0;
-    margin: -1px;
+.popup_icons a {
+  text-decoration: none;
```

Comment:
Add `display: block` to the link, to the link, it indicates to the user more obvious focused element:
Without `display: block`:
![Screenshot_1](https://user-images.githubusercontent.com/16196199/79698513-19541b80-8292-11ea-9b79-042950e94930.png)

With `display: block`:
![Screenshot_2](https://user-images.githubusercontent.com/16196199/79698512-18bb8500-8292-11ea-9b83-17492d6714b3.png)

Reply:
Thank you very much, I honestly did not know what to do with it =)